docs/en/02_Developer_Guides/14_Files/02_Images.md

```diff
@@ -200,6 +200,14 @@ SilverStripe\Core\Injector\Injector:
       Quality: 90
 ```
 
+### Lazy Loading
+
+Images will be [lazy loaded](https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading) by default.
+
+To disable lazy loading for an individual image, you can call `$MyImage.LazyLoad(false)`.
```

Comment:
Specify that this is only for templates

Reply:
What do you mean?

You could technically call this via PHP as well.
Rendering-wise it currently only works with templates, but that's kind of obvious?

Reply:
You could make a quick mention that images inserted via the WYSIWYG are handled differently i.e. you cannot disable lazyloading on them by calling LazyLoad(false)

I know it's seems kind of obviousus, though we have plenty of context on this whole thing, whereas other developers reading these docs probably will not